packages/babel-plugin/src/babel-plugin.ts

```diff
@@ -87,6 +90,11 @@ export default declare<State>((api) => {
               }
             }
           }
+
+          if (/(x|X)css={/.exec(file.code)) {
+            // xcss prop was found turn on Compiled
+            state.compiledImports = {};
```

Comment:
Turn on xcss prop if we have found prop usage.

Reply:
Why not lean in on how the CSS prop enables compiled with the above logic? Or check for the existence in `JSXOpeningElement`?

Reply:
Because when someone is using the inline object exclusively:

```
<Component xcss={{ color: 'red' }} />
```

Having to turn on the jsx pragma or import Compiled APIs would be a point of friction.

I can go both ways for turning it on with the regex check or existence in JSXOpeningElement. What would the team prefer?

Reply:
JSX element is more consistent with css prop, but @JakeLane can make the call

Reply:
Alright will catch up. Both are valid IMO – not needing it if you're exclusively only using xcss prop is great for just one less thing to remember to do.

Reply:
This is dicey because prop naming isn't required for `xcss` to work, we'd be leaning on convention here. I think I'd prefer to keep build perf here ideal so we can rely on plain text.

Reply:
> This is dicey because prop naming isn't required for xcss to work

In what sense? Currently / by design inline object compilation will only work on a prop called `xcss` or `yzXcss`.

Reply:
meaning we could turn on the API in numerous ways, rather than just looking at the prop name